app/src/main/java/net/squanchy/tweets/domain/view/TweetViewModel.kt

```diff
@@ -3,6 +3,7 @@ package net.squanchy.tweets.domain.view
 import net.squanchy.support.lang.Optional
 import net.squanchy.tweets.domain.TweetLinkInfo
 
+@Suppress("LongParameterList") // This is just a big model - TODO refactor this to split it up
```

Comment:
increase the thresholds (I'm marking all them so it's easier to find them and see if we did all of them)